test/c/CMakeLists.txt

```diff
@@ -1,4 +1,27 @@
+if (NOT MSVC)
+	set (CMAKE_PREFIX_PATH "$ENV{BOOST_ROOT}")
```

Comment:
yeah this is probably a better idea even though most people in windows from what I read seem to set this environment variable.

But to be honest .. the best thing would be to use the same thing you made for the ETHCmakeDependecies in cpp-ethereum and have it download boost and add its root as the `CMAKE_PREFIX_PATH`.

We need to make it as automatic as possible in windows